docs/src/modules/components/MarkdownDocs.js

```diff
@@ -141,6 +141,7 @@ export default function MarkdownDocs(props) {
                 jsxPreview: demo.jsxPreview,
                 rawTS: demo.rawTS,
                 tsx: demoComponents[demo.moduleTS] ?? null,
+                gaLabel: fileNameWithLocation.replace(/^\/docs\/data\//, ''),
```

Comment:
Would this work with Windows?

Reply:
I have checked. We are good, it's taken care of in:

https://github.com/mui/material-ui/blob/5d387c0e588c163a0c8c159ab53243b9d895a10b/packages/markdown/loader.js#L119-L122